index.html

```diff
@@ -0,0 +1,124 @@
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+          <!---The Third card -->
+          <div class="flex">
+            <img src="images/card3-image.png" alt="Image for Card 3">
+            <h2>Multi-Post
+                Stories</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+           <!---The Fourth card -->
+           <div class="flex">
+            <img src="images/card4-image.png" alt="Image for Card 4">
+            <h2>TONIC</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+
+    </section>
+    <!--About section-->
+    <section class="about">
+        <div class="about-content">
+            <h2>About <br> Myself</h2>
+            <p class="my-intro">Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what
+                 you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <br>
+            <p class="lets-connect-message">LET'S CONNECT</p>
+            <ul class="connect-media">
+                <li><a  href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a  href="#"><img src="images/Linkedinicon.svg" alt="linkedin"></a></li>
+                <li><a  href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a  href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a  href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            </ul>
+            <button class="see-more resume">Get my resume</button>
+        </div>
+        <ul class=" about-list">
```

Comment:
- [ ] Nice working on your language section, kindly adjust the `width` and `margin` of the language list so it matches the Figma Template

### Your Design
![is-figma-lang](https://user-images.githubusercontent.com/48371156/176655309-0061f44e-36c8-4d26-af34-18213f532a1a.png)

### Figma Design
![is-work-lang](https://user-images.githubusercontent.com/48371156/176655319-6d524644-9a0f-40b7-bde3-cab8a410a466.png)